package.json

```diff
@@ -27,6 +27,7 @@
     "@babel/plugin-proposal-private-methods": "^7.18.6",
     "@babel/plugin-transform-runtime": "^7.25.9",
     "@babel/preset-env": "^7.25.9",
+    "@ministryofjustice/frontend": "^2.2.4",
```

Comment:
Is this being used yet? If not I would suggest removing it to reduce the size of this PR.